pgp/Cargo.toml

```diff
@@ -13,7 +13,19 @@ keywords = ["pgp", "openpgp", "encrypt", "decrypt", "sign"]
 
 [package.metadata.docs.rs]
 all-features = true
-rustdoc-args = ["--cfg", "docsrs"]
+rustdoc-args = ["--cfg", "docs_rs"]
+
+[features]
+default = ["http"]
```

Comment:
I find the name `http` hard to understand in the PGP context. Maybe something like `discovery`, `key-discovery`  or `pubkey-discovery` would fit better, what do you think? Modules could also move to a dedicated submodule, sth like `wkd.rs` → `discovery/wkd.rs`, `hkp.rs` → `discovery/hkp.rs` and `http.rs` → `discovery/mod.rs`.

Reply:
Used `key-discovery`. I didn't move everything to a module, I don't know the structure well enough to know what makes sense, but feel free to add a commit to this branch or after merging :)